Add contributors portal and other links (#1)
artefactual/.github: profile/README.md

# Artefactual

For more information about us, please [visit artefactual.com](https://www.artefactual.com/).

## Contributing

We are actively developing the [Contributors portal](https://contributors.artefactual.com) to host guidelines and information

> related to contributing to AtoM, Archivematica, and the dependencies which we maintain.

Reviewer: we don't maintain the dependencies so i might suggest rewording this to something like, 

* "and their dependencies" or
* "as well as listing their dependencies"

Also you mention Enduro on the actual portal, but not here - not sure if that's intentional or not? One option is simply to not list projects at all (since other applications like AIPScan etc could end up here as well), and just say "the projects that Artefactual maintains" or something

> related to contributing to AtoM, Archivematica, and the dependencies which we maintain.

Author: ```suggestion
related to contributing to the projects that Artefactual maintains.
```

## Related organizations

Most of our projects are hosted at one of the following:

- [Main Artefactual repo](https://www.github.com/artefactual), this page
- [Artefactual labs](https://www.github.com/artefactual-labs), smaller projects and experiments

> - [Archivematica repo](https://www.github.com/archivematica), for related issues and some projects

Author: ```suggestion
- [Archivematica repo](https://www.github.com/archivematica), for related issues and some projects
- [Artefactual-SDPS](https://github.com/artefactual-sdps) for projects related to Enduro
```

> - [Archivematica repo](https://www.github.com/archivematica), for related issues and some projects

Reviewer: if you wanted you could add [Artefactual-SDPS](https://github.com/artefactual-sdps) for projects related to Enduro
